tests/e2e/mocha-java-vertx.opts

```diff
@@ -3,5 +3,6 @@
 -u tdd 
 --bail
 --full-trace
---spec dist/tests/e2e/JavaVertx.spec.js
```

Comment:
I have a suggestion about this approach, but it may be changed in the future.

Reply:
@Ohrimenko1988 And what the suggestion is? :D